peachjam/adapters/gazettes.py

```diff
@@ -79,6 +86,12 @@ def update_document(self, expression_frbr_uri):
             if ga_gazette.locality:
                 data["locality"] = Locality.objects.get(code=ga_gazette.locality.code)
 
+            if ga_gazette.nature:
+                data["nature"] = DocumentNature.objects.get_or_create(
+                    code=ga_gazette.nature.code,
+                    defaults={"name": ga_gazette.nature.name},

```

Comment:
Can you treat this like you do in indigo: split on '-' and use `.capitalize()`

Reply:
changed